.eslintrc.js

```diff
@@ -248,7 +248,7 @@ module.exports = {
         "padded-blocks": "off",
         "padding-line-between-statements": "error",
         "prefer-arrow-callback": "error",
-        "prefer-const": "off",
+        "prefer-const": "error",
```

Comment:
we should use 'const' for variables that we don't intend to override - this setting ensures that (https://eslint.org/docs/rules/prefer-const)